### DISCUSSION TITLE
RPi v1.1 baremetal not working?

### DISCUSSION THREAD

**Discussion by raerek** [nesbox/TIC-80#2359] [General]
Hi!

Could somebody help me in running a baremetal RPi TIC80?

I've FAT32-formatted a microSD card, and copied the files form the current release tic80-v1.1-rpi-baremetal.zip. Tried on an RPi3, but all I get is four colored rectangles. 

Copied the files form tic80-v1.1-rpi4-baremetal.zip and tried in a RPi400, and same thing happens, i.e. same four rectangles.

Using the file [here](https://github.com/nesbox/TIC-80/issues/1143#issuecomment-668239349) I was able to have a running TIC80 on the RPi3, but not on the RPi400. For me this proves that the card formatting and the card iself are okay.

Do I do something wrong or is the baremetal build broken?
I've tried to follow [these](https://github.com/nesbox/TIC-80/blob/main/build/baremetalpi/README.md) steps, but unfortunately the readme seems to be missing the cloning of circle-stdlib and later during compiling I've run into a mismatching definition of variables. I can get back with the exact error message, but for me it seems that the build has a CI pipeline set up so I think I am doing something wrong.

So someone help me please.

**Comment by raerek** [comment]
Could these be related? 
https://github.com/nesbox/TIC-80/issues/1931
https://github.com/nesbox/TIC-80/issues/1820

Or should I open a new issue?

**Reply by nesbox** [reply]
Maybe just update this issue https://github.com/nesbox/TIC-80/issues/1820
